cmake/pthreadpool-config.cmake.in

```diff
@@ -0,0 +1,5 @@
+@PACKAGE_INIT@
```

Comment:
I believe this also needs the following prior to this line to make sure `FXdiv` is present:
```suggestion
include(CMakeFindDependencyMacro)

find_dependency(FXdiv)

@PACKAGE_INIT@
```